lib/charliecloud.py

```diff
@@ -481,6 +485,10 @@ def cmd(argv, fail_ok=False, **kwargs):
    """Run command using cmd_base(). If fail_ok, return the exit code whether
       or not the process succeeded; otherwise, return (zero) only if the
       process succeeded and exit with fatal error if it failed."""
+   if (log_quiet > 1):
+      kwargs["stdout"] = subprocess.DEVNULL
+   if (log_quiet == 3):
+      kwargs["stderr"] = subprocess.DEVNULL
```

Comment:
I think this will be mooted by other changes, but this will be more clear if the two conditions use the same comparison, e.g.:
```suggestion
   if (log_quiet > 1):
      kwargs["stdout"] = subprocess.DEVNULL
      if (log_quiet > 2):
         kwargs["stderr"] = subprocess.DEVNULL
```